src/vivarium_conic_lsff/components/observers.py

```diff
@@ -132,4 +132,72 @@ def get_transition_count(pop, config, disease, transition, event_time, age_bins)
                                                         year=event_time.year)
     base_filter = QueryString('')
     transition_count = get_group_counts(transitioned_pop, base_filter, base_key, config, age_bins)
-    return transition_count
\ No newline at end of file
+    return transition_count
+
+
+
+class LiveBirthWithNTDObserver:
+    """Observes births and births with neural tube defects. Output can be stratified
+    by year and by sex.
+    """
+    configuration_defaults = {
+        'metrics': {
+            'live_births_with_ntds': {
+                'by_year': True,
+                'by_sex': True,
+            }
+        }
+    }
+
+    @property
+    def name(self):
+        return f'live_birth_with_ntd_observer'
+
+    def setup(self, builder):
+        self.disease = project_globals.NTD_MODEL_NAME
+        self.config = builder.configuration['metrics']['live_births_with_ntds'].to_dict()
+        self.live_birth_counts = Counter()
+        self.with_disease = Counter()
+        tmp = builder.configuration['time']['start']
+        self._step_start_time = pd.Timestamp(tmp['year'], tmp['month'], tmp['day'])
```

Comment:
Still think unpacking builder.configuration.time.start here directly.  It's why the keys are structured the way they are and allows us the flexibility to specify any args a timestamp will take without code changes.